src/test/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/DisabledCategoriesTest.java

```diff
@@ -209,6 +209,8 @@ protected void logAll(AuditLog auditLog) {
 		logTransportBadHeaders(auditLog);
 		logTransportFailedLogin(auditLog);
 		logTransportSucceededLogin(auditLog);
+
```

Comment:
nit: empty line

Reply:
There are new line separations above it hence added.